### ISSUE TITLE
Add remaining contract interaction functionalities in tasit-action

### ISSUE THREAD

**Issue by marcelomorgado** [tasitlabs/TasitSDK#75]
#40 open conversation threads as well:

- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r243329206
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r243331285
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r243333073
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r243742808
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r243742850
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r243743035
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r243743061
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244824045
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244824391
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244824688
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244826315
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244826599
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244827784
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244828122
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244828219
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244828773
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244828910
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244832841
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244833417
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244833858
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244834096
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244834155
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244834506
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244834752
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244834991
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244835367
- https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244835816

**Comment by pcowgill** [comment]
Also:
https://github.com/tasitlabs/TasitSDK/pull/71/files/f903f97fd04ecd7797a964549bde6db8e06e23b2..83f1105b218fb6779dc49cd6a336e2de7f0baad6#r244863508
https://github.com/tasitlabs/TasitSDK/pull/71/files/f903f97fd04ecd7797a964549bde6db8e06e23b2..83f1105b218fb6779dc49cd6a336e2de7f0baad6#r244863774

**Comment by pcowgill** [comment]
Also we still need to expose `tasit-action` in the `tasit-sdk` parent package

**Comment by pcowgill** [comment]
We can close this issue when enough of the threads linked to above are resolved (since _all_ remaining functionality is potentially a moving target)

**Comment by marcelomorgado** [comment]
Right. I've reviewed all threads and updated them.
There are a few open questions (almost all answering if the related thread can be resolved).

**Comment by pcowgill** [comment]
@marcelomorgado Are there still #40 open conversation threads as well?

**Comment by pcowgill** [comment]
@marcelomorgado FYI I responded to all of these:
https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r243742808 [waiting explanation]
https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244824391 [waiting response]
https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244826315 [waiting response]
https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244826599 [waiting response]
https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244828122 [waiting response]
https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244828910 [waiting response]
https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244834096 [waiting response]
https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244834506 [waiting response]
https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244834752 [waiting response]
https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244834991 [waiting response]
https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244835816 [waiting response]
https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r243743035 [waiting response]

and this one

https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r244863508 [waiting response]

**Comment by marcelomorgado** [comment]
> @marcelomorgado Are there still #40 open conversation threads as well? (I ask because it says that in the description for this issue.)

No, not really. #40 was the issue that was the target of PR [#71](https://github.com/tasitlabs/TasitSDK/pull/71) that have his threads being track from here. :laughing: 
Very messy I know. I'll trying to do references more clear on next times.

**Comment by pcowgill** [comment]
> No, not really. 40 was the issue that was the target of PR 71 that have his threads being track from here. 😆
Very messy I know. I'll trying to do references more clear on next times.

Ah, got it. No problem! Given the number of topics we’re tracking you’ve been doing an impressive job of being clear already. Thanks!

**Comment by marcelomorgado** [comment]
Thank you.

**Comment by marcelomorgado** [comment]
Could I resolve this?
